src/app/components/Text/__snapshots__/index.test.jsx.snap

```diff
@@ -6,15 +6,6 @@ exports[`Text should render bold 1`] = `
   font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
   padding-bottom: 1rem;
   margin: 0;
-  font-size: 0.9375em;
```

Comment:
These shouldn't have been removed from the snapshot, following [psammead#164](https://github.com/BBC-News/psammead/pull/164). I think it's a case of the cached version of the Psammead package being used -- bumping the requested version in `package.json` immediately causes these snapshots to fail.